src/main/java/com/example/demo/NamesGetController.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import java.util.List;
+
+@RestController
+public class NamesGetController { //何をGetするControllerかわかるように
+
+    @GetMapping("/names") //扱うリソースの複数形
+    public List<String> getNames() {
+        return List.of("mori", "kawano"); //"kawano"前のスペース
```

Comment:
変更履歴はコミットログから確認できます。
https://github.com/miya2211/kadai07/pull/1/commits
なのでソースコード内には基本的にはこういった何を修正したか？のコメントは不要ですね。
```suggestion
        return List.of("mori", "kawano");
```

エンジニアはコミットログも確認する機会が多いです。
今回のようなレビューやバグが判明したときにどのコミットでバグが発生しているのか追跡したりするときですね。
なのでコミットログに何を書くかはとても大事ですのでぜひ一度調べてみてください。
`コミットログ なにを書く`
`what to write commit message`
などで調べると参考情報が出てくると思います。

Reply:
ご指摘いただいた点修正いたしました。
コミットログについて調べます！